docs/get-started/bridging.mdx

```diff
@@ -71,6 +71,21 @@ The request includes the tez to bridge, the address of the Etherlink Smart Rollu
 1. The bridge contract forwards the ticket to the stored Etherlink Smart Rollup address and clears its storage for the next transfer.
 1. The Etherlink Smart Rollup uses the information in the ticket to mint tokens on Etherlink and send them to the user's Etherlink address.
```

Comment:
We could add an additional line saying that a deposit on L1 creates a transfer on L2 from the zero address: https://explorer.etherlink.com/address/0x0000000000000000000000000000000000000000